cyclops-ui/src/components/pages/TemplateStore/styles.module.css

```diff
@@ -17,3 +17,21 @@
 .edittemplate:hover {
   transform: scale(1.4);
 }
+
+.statustemplate {
+  color: grey;
+  transform: scale(1.2);
+  transition: transform 0.2s;
+}
+
+.loading {
+  color: orange;
+}
```

Comment:
I think this is not used

Reply:
Yes that's right!! seems like during testing different solutions I must have forgot to remove this